src/Common/FitOnFhir.Common.Tests/AuthStateServiceTests.cs

```diff
@@ -136,61 +142,34 @@ public void GivenRedirectUrlNotInApprovedList_WhenCreateAuthStateCalled_ThrowsRe
             SetupHttpRequest(ExpectedToken);
             AuthConfiguration.ApprovedRedirectUrls.Returns(new List<string>());
 
-            Assert.Throws<RedirectUrlException>(() => _authStateService.CreateAuthState(Request));
-
-            _logger.Received(1).Log(
-                Arg.Is<LogLevel>(lvl => lvl == LogLevel.Error),
-                Arg.Is<string>(msg =>
-                    msg == "The redirect URL was not found in the list of approved redirect URLs."));
-
-            _logger.DidNotReceive().Log(
-                Arg.Is<LogLevel>(lvl => lvl == LogLevel.Error),
-                Arg.Is<string>(msg =>
-                    msg == $"The required parameter {Constants.RedirectUrlQueryParameter} was not provided in request"));
+            Assert.Throws<AuthStateException>(() => _authStateService.CreateAuthState(Request));
         }
 
         [Fact]
         public void GivenRedirectUrlNotInRequest_WhenCreateAuthStateCalled_ThrowsRedirectUrlExceptionAndErrorIsLogged()
```

Comment:
It looks like the condition in the test name is no longer valid, as we don't log anymore.